src/senselab/audio/workflows/transcribe_timestamped/transcribe_timestamped.py

```diff
@@ -0,0 +1,109 @@
+"""Transcribes audio files with timestamps."""
+
+# TODO: Please double-check this because tests are failing
+from typing import List, Tuple
+
+import pydra
+
+from senselab.audio.data_structures import Audio
+from senselab.audio.tasks.forced_alignment.constants import SAMPLE_RATE
+from senselab.audio.tasks.forced_alignment.forced_alignment import (
+    align_transcriptions,
+)
+from senselab.audio.tasks.preprocessing import (
+    downmix_audios_to_mono,
+    resample_audios,
+)
+from senselab.audio.tasks.speech_to_text import transcribe_audios
+from senselab.utils.data_structures import HFModel, Language, ScriptLine
+from senselab.utils.tasks.batching import batch_list
+
+
+def transcribe_timestamped(
+    audios: List[Audio],
+    model: HFModel = HFModel(path_or_uri="openai/whisper-tiny"),
+    language: Language = Language(language_code="en"),
+    n_batches: int = 1,
+) -> List[List[ScriptLine]]:
+    """Transcribes audio files and timestamps them with forced alignment.
+
+    This function processes the given list of Audio objects by performing
+    necessary preprocessing steps (such as downmixing channels and resampling),
+    transcribes the audio using the specified speech-to-text model, and applies
+    forced alignment to generate a list of ScriptLine objects with timestamps.
+
+    Args:
+        audios (list[Audio]): List of Audio objects to be transcribed and
+                              timestamped.
+        model (HFModel, optional): A Huggingface model for speech-to-text.
+                                   Defaults to 'whisper'.
```

Comment:
You should specify that this defaults to "whisper-tiny"

Reply:
Also the type hints in the function declaration don't specify that these three arguments are optional